### PR TITLE
Adding Symfony 4 support

### PR DESCRIPTION
**Pull request by bocharsky-bw**
#SymfonyConHackday2017

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,11 +36,13 @@ matrix:
       env: SYMFONY_VERSION='3.2.*'
     - php: 7.0
       env: SYMFONY_VERSION='3.3.*'
-
+    - php: 7.1
+      env: SYMFONY_VERSION='4.0.*' DEPENDENCIES='beta'
```
Comment (xabbuh) [inline .travis.yml]: Symfony 4 is stable now

Comment (bocharsky-bw) [reply .travis.yml]: Thanks for this ping, Christian! I'll tweak it

**Comment by xabbuh** [comment]
Actually, tests for Symfony 4 do not run (see #122 to unlock that).

**Comment by bocharsky-bw** [comment]
Good catch, thanks! Waiting for your PR to be merged

**Comment by Valegar** [comment]
Thanks for the patch! Now we just need a new release (v1.0.7 maybe?) to be able to use this bundle as stable package with Symfony 4...

**Pull request merged**